src/Libraries/CoreNodeModels/Enum.cs

```diff
@@ -69,11 +69,21 @@ protected AllChildrenOfType() : base(outputName)
             RegisterAllPorts();
         }
 
+        protected AllChildrenOfType(string value) : base(value)
+        {
+            RegisterAllPorts();
+        }
+
         [JsonConstructor]
         protected AllChildrenOfType(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(outputName, inPorts, outPorts)
         {
         }
 
+        [JsonConstructor]
+        protected AllChildrenOfType(string value, IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(value, inPorts, outPorts)
```

Comment:
hmm - I'm not sure how this constructor will ever be invoked by the deserializer - since the `outputName` field doesn't get serialized I don't think anything will ever be passed here - also naming it `value` is probably a good way to confuse `json.net` - since there might be some other property called `value` that is serialized.

The second issue I think needs to be tested, is since the output name is not serialized, what happens when you reopen a graph containing one of these nodes that was created with a different outputName, does it deserialize correctly with the right name?

Reply:
If what I said makes no sense, give this a read about how `json.net` uses these json constructors:
https://stackoverflow.com/questions/23017716/json-net-how-to-deserialize-without-using-the-default-constructor

Reply:
Thank you for your reply, Michael. Here is a comparison of the old and new nodes in the script, and does not affect the function. 
![test](https://user-images.githubusercontent.com/33445445/96821825-4b218700-145b-11eb-9b1f-2c2ac2a5161f.png)
I will update this variable name to avoid Json confusion.

Reply:
Hello @ZiyunShang - one comment: The term 'Class' is not familiar to the majority of our user base inside of Dynamo, where as "Type" is familiar coming from the Revit context. It may make it more difficult for users to understand.

Reply:
Hi @Amoursol , in RevitAPI, there is a type of Element called "ElementType", which is confusing with this "Type". And after discussing with my team, I think that 'Class' is more in line with the output of this node.